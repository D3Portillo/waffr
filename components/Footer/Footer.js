/* eslint-disable @next/next/no-img-element */
import Button from "@/components/Button";

function Footer() {
  return (
    <div className="flex pb-6 pt-24 items-end lg:items-center">
      <div className="flex lg:items-center flex-wrap space-y-2 lg:space-y-0 lg:space-x-4 flex-col lg:flex-row">
        <Button
          isExternal
          isLink
          href="https://github.com/D3Portillo/waffr/wiki"
          className="!p-0"
        >
          ROADMAP
        </Button>

        <Button isExternal isLink href="https://getwaffle.io/" className="!p-0">
          FRAMEWORK
        </Button>
        <Button
          href="https://www.d3portillo.me/"
          isExternal
          isLink
          className="!p-0"
        >
          DEVELOPER
        </Button>
      </div>
      <div className="flex flex-grow" />
      <div className="px-6">
        <img className="h-12" alt="WAFFR LOGO" src="/text-logo.svg" />
      </div>
    </div>
  );
}

export default Footer;
