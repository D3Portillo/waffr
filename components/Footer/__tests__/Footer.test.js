import { render } from "@testing-library/react";
import Footer from "../Footer";

describe("Footer", () => {
  let node;
  afterEach(() => {
    if (node && node.unmount) {
      node.unmount();
    }
    jest.clearAllMocks();
  });

  it("matches snapshot", () => {
    node = render(<Footer />);
    expect(node.container).toMatchSnapshot();
  });
});
