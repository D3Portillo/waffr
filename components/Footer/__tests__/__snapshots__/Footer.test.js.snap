// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<div>
  <div
    class="flex pb-6 pt-24 items-end lg:items-center"
  >
    <div
      class="flex items-start lg:items-center flex-wrap space-y-2 lg:space-y-0 lg:space-x-4 flex-col lg:flex-row"
    >
      <a
        class="!p-0 flex cursor-pointer justify-center items-center uppercase font-bold px-8 py-4 rounded-lg space-x-2 text-white opacity-70 hover:opacity-100"
        href="https://github.com/D3Portillo/waffr/wiki"
        rel="noopener noreferrer"
        target="_blank"
      >
        <div>
          ROADMAP
        </div>
      </a>
      <a
        class="!p-0 flex cursor-pointer justify-center items-center uppercase font-bold px-8 py-4 rounded-lg space-x-2 text-white opacity-70 hover:opacity-100"
        href="https://getwaffle.io/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <div>
          FRAMEWORK
        </div>
      </a>
      <a
        class="!p-0 flex cursor-pointer justify-center items-center uppercase font-bold px-8 py-4 rounded-lg space-x-2 text-white opacity-70 hover:opacity-100"
        href="https://www.d3portillo.me/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <div>
          DEVELOPER
        </div>
      </a>
    </div>
    <div
      class="flex flex-grow"
    />
    <div
      class="px-6"
    >
      <img
        alt="WAFFR LOGO"
        class="h-12"
        src="/text-logo.svg"
      />
    </div>
  </div>
</div>
`;
